src/Services/Page.vala

```diff
@@ -218,19 +243,24 @@ public class ENotes.PageTable : DatabaseTable {
             page.subtitle = "";
         }
     }
-
-    private const string[] SYMBOLS_BLACKLIST = {"#", "`", "\t", "<br>", ">", "<", "\n", "~" };
+      
     private string cleanup (string line) {
         string output = line;
 
-        if (line.contains ("---")) return "";
-
-        foreach (string item in SYMBOLS_BLACKLIST) {
-            if (output.contains (item)) {
-                output = output.replace (item, "");
+        try {
+            foreach (BLMember item in Regex_complex_commands) {
+                  output = item.reg.replace (output, -1, 0, item.replace);        
```

Comment:
Check your white spaces here :) You seem to have gotten 6 instead of 4

Reply:
done